### ISSUE TITLE
Can't get this working

### ISSUE THREAD

**Issue by HAPSagan** [krissen/pollenprognos-card#3]
Do I have to delete the card I had before ("Lovelace Pollen Prognos Card") as they seem to have the same name and the previous one shows up when I try to add this to the card? Both seem to be added with 'custom:pollenprognos-card'.

**Comment by HAPSagan** [comment]
I mean 'custom:pollenprognos-card'

**Comment by Mpgod80** [comment]
I have the same issue with this as HAPSagan =/

**Comment by Klangen82** [comment]
Same here, the old look will not disappear. Cleared cache on the browser but still see the old look...

**Comment by andjo** [comment]
Yes, they are in conflict about `custom:pollenprognos-card`. Find the other frontend repository and remove it.

**Comment by Klangen82** [comment]
I did this of course but did not help. To get it working I removed all that has to do with Pollenprognos card then i needed to reboot my hardware (not just a standard reboot) then I cleared the cache and added the Pollenprognos card via HACS. After this it work :) So all good from my side!

**Comment by krissen** [comment]
A full reboot is not required. However, in HA, both a reload of components *and* a refresh is needed. (Both are found in the hamburger menu of HA, top-right.)
